R/helpers-misc.R

```diff
@@ -0,0 +1,79 @@
+#' Select all variables or the last variable
+#'
+#' @description
+#'
+#' These functions are [selection helpers][select_helpers].
+#'
+#' * [everything()] selects all variable. It is also useful in
+#'   combination with other tidyselect operators.
+#'
+#' * [last_col()] selects the last variable.
+#'
+#' @inheritParams starts_with
```

Comment:
Use `@family selection helpers` to cross-link these all together?

Reply:
I find it a bit confusing because it doesn't show the aliases:

```
Other selection helpers: all_of(), everything(), starts_with()
```